docs/resources/rds_sql_audit.md

```diff
@@ -0,0 +1,63 @@
+---
+subcategory: "Relational Database Service (RDS)"
+---
+
+# huaweicloud_rds_sql_audit
+
+Manages RDS sql audit resource within HuaweiCloud.
+
+-> **NOTE:** Only Mysql and PostgreSQL is supported.
+
```

Comment:
missing description: one instance can manage only one audit resource.

Reply:
it seems useless，there are many resources that can only be created once under one resource, so maintaining consistency is more reasonable

Reply:
WARN: the instance ID is used in your resource ID design